middleware/auth-utils/grant-manager.js

```diff
@@ -424,7 +426,7 @@ GrantManager.prototype.validateToken = function validateToken (token, expectedTy
     } else if (token.content.iat < this.notBefore) {
       reject(new Error('invalid token (stale token)'));
     } else if (token.content.iss !== this.realmUrl) {
-      reject(new Error('invalid token (wrong ISS)'));
+      reject(new Error(`invalid token (wrong ISS): issuer on token is ${token.content.iss} but realmUrl is ${this.realmUrl}`));
```

Comment:
As I recall, it doesn't. I wanted to log it but thought I'd have to integrate the outlier logger and didn't want to do that. Or did you mean just console log? I can also just remove it since it served it's purpose.

Reply:
I think you're right that it's useful information for the logs. Even in the longer term, I think it could be valuable. Agreed that integrating the outlier logger isn't a great option. I was thinking a `console.log` would be fine, but if that's unnecessary now, then maybe it's easier to revert this.

Reply:
Ended up just removing the extra message info.